docs/CONFIGURATION.md

```diff
@@ -1,3 +1,4 @@
+# Configuration #
```

Comment:
Do we want to use the format with opening and closing header markdown going forward? `# header #` vs `# header`

It is very minor but I am for consistency.